py3status/modules/bluetooth.py

```diff
@@ -28,11 +28,6 @@
 """
 
 import re
-import shlex
-
-from subprocess import check_output
-
-BTMAC_RE = re.compile(r'[0-9A-F:]{17}')
```

Comment:
yeah it's all judgement calls, but the lighter each update is the better

Thanks